lib/101/statefull_life_cycle.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+class StatefullLifeCycleLearn extends StatefulWidget {
+  const StatefullLifeCycleLearn({super.key, required this.message});
+
+  final String message;
+
+  @override
+  State<StatefullLifeCycleLearn> createState() =>
+      _StatefullLifeCycleLearnState();
+}
+
+class _StatefullLifeCycleLearnState extends State<StatefullLifeCycleLearn> {
+  String _message = " ";
+  late final bool _isOdd;
+
+  @override
+  void initState() {
+    super.initState();
+    _message = widget.message;
+    _isOdd = widget.message.length.isOdd;
+
+    _computeName();
+  }
+
+  void _computeName() {
+    if (_isOdd) {
+      _message += " Çift";
+    } else {
+      _message += " Tek";
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: _isOdd ? const Text("Kelime tek") : const Text("Kelime çift"),
```

Comment:
Since these two widgets are exactly the same and only difference is text, this condition can be simplified as follows:

```suggestion
        title: Text(_isOdd ? "Kelime tek" : "Kelime çift"),
```

Reply:
Done.